plugins/woocommerce/tests/php/includes/wc-order-functions-test.php

```diff
@@ -184,4 +184,69 @@ public function test_wc_update_coupon_usage_counts() {
 		$this->assertEquals( 1, $order->get_data_store()->get_recorded_coupon_usage_counts( $order ) );
 		$this->assertEquals( 1, ( new WC_Coupon( $coupon ) )->get_usage_count() );
 	}
+
+	/**
+	 * Test getting total refunded for an item with and without refunds.
+	 */
+	public function test_get_total_refunded_for_item() {
+		// Create a product.
+		$product = WC_Helper_Product::create_simple_product();
+		$product->set_regular_price( 99.99 );
+		$product->save();
+
+		// Create an order with the product.
+		$order = new WC_Order();
+		$item  = new WC_Order_Item_Product();
+		$item->set_props(
+			array(
+				'product'  => $product,
+				'quantity' => 2,
+				'total'    => 199.98,
```

Comment:
To satisfy my own curiosity even when we set `float` values here, `get_total()` returns these as `string` values?

Reply:
For order items yes, they are stored and retrieved as strings.